src/gui/transferlistmodel.cpp

```diff
@@ -224,12 +225,15 @@ QVariant TransferListModel::headerData(const int section, const Qt::Orientation
             case TR_AMOUNT_LEFT: return tr("Remaining", "Amount of data left to download (e.g. in MB)");
             case TR_TIME_ELAPSED: return tr("Time Active", "Time (duration) the torrent is active (not paused)");
             case TR_SAVE_PATH: return tr("Save path", "Torrent save path");
+            case TR_DOWNLOAD_PATH: return tr("Incomplete save path", "Torrent incomplete save path");
             case TR_COMPLETED: return tr("Completed", "Amount of data completed (e.g. in MB)");
             case TR_RATIO_LIMIT: return tr("Ratio Limit", "Upload share ratio limit");
             case TR_SEEN_COMPLETE_DATE: return tr("Last Seen Complete", "Indicates the time when the torrent was last seen complete/whole");
             case TR_LAST_ACTIVITY: return tr("Last Activity", "Time passed since a chunk was downloaded/uploaded");
             case TR_TOTAL_SIZE: return tr("Total Size", "i.e. Size including unwanted data");
             case TR_AVAILABILITY: return tr("Availability", "The number of distributed copies of the torrent");
+            case TR_INFOHASH_V1: return tr("InfoHash v1", "i.e: torrent info hash v1");
+            case TR_INFOHASH_V2: return tr("InfoHash v2", "i.e: torrent info hash v2");
```

Comment:
```suggestion
            case TR_INFOHASH_V1: return tr("Info Hash v1", "i.e: torrent info hash v1");
            case TR_INFOHASH_V2: return tr("Info Hash v2", "i.e: torrent info hash v2");
```
Because that's how they appear in General tab.

Reply:
You're right!